test/abstractarray.jl

```diff
@@ -1,3 +1,4 @@
+using Core: Argument
```

Comment:
```suggestion
```

Reply:
That's weird. VS Code seems to be inserting random usings automatically? It keeps happening.